ts/features/itwallet/common/components/ItwDiscoveryBanner.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  Banner,
+  IOVisualCostants,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import React from "react";
+import { View } from "react-native";
+import I18n from "../../../../i18n";
+import { useIONavigation } from "../../../../navigation/params/AppParamsList";
+import { useIOSelector } from "../../../../store/hooks";
+import { isTrialActiveSelector } from "../../../trialSystem/store/reducers";
+import { ITW_ROUTES } from "../../navigation/routes";
+import { ITW_TRIAL_ID } from "../utils/itwTrialUtils";
+import { itwLifecycleIsValidSelector } from "../../lifecycle/store/selectors";
+
+type ItwDiscoveryBannerProps = {
+  withTitle?: boolean;
+  ignoreMargins?: boolean;
+};
+
+export const ItwDiscoveryBanner = ({
+  withTitle = true,
+  ignoreMargins = false
+}: ItwDiscoveryBannerProps) => {
+  const bannerRef = React.createRef<View>();
+  const navigation = useIONavigation();
+  const [isVisible, setVisible] = React.useState(true);
+  const isItwTrialActive = useIOSelector(isTrialActiveSelector(ITW_TRIAL_ID));
+  const isItwValid = useIOSelector(itwLifecycleIsValidSelector);
+
+  // Banner should be hidden if:
+  if (
+    !isVisible || // The user closed it by pressing the `x`
+    !isItwTrialActive || // The user is not part of the trial
```

Comment:
Looking to the selector this condition is false because trial status equal to ACTIVE. Should it be equal to SUBSCRIBED? With these conditions I cannot display the banner in the message screen.

Reply:
`SUBSCRIBED` means that the user requested the trial but it is not active yet, while `ACTIVE` mean that the user is effectively enabled to use the trial. So I think `ACTIVE` is appropriate for this use case.